src/jqconsole.coffee

```diff
@@ -439,10 +439,18 @@ class JQConsole
   #   @arg continue_label: The new continuation label for the next prompt. Optional.
   SetPromptLabel: (main_label, continue_label) ->
     @prompt_label_main = main_label
+    prompt_selector = '>span+span>span:first-child'
+    $('.' + CLASS_PROMPT + prompt_selector).text(main_label);
     if continue_label?
       @prompt_label_continue = continue_label
     return undefined
 
+  # Update the main prompt label.
+  UpdatePromptLabel: ->
+    prompt_selector = '>span+span>span:first-child'
+    full_selector = '.' + CLASS_PROMPT + prompt_selector
+    $(full_selector).text(@prompt_label_main)
```

Comment:
Let's scope this to `@$console` otherwise we can unexpectedly change other console on the page.